pyproject.toml

```diff
@@ -27,8 +27,8 @@ packages = [
 python = "^3.7.1"
 nautobot-chatops = "^1.1.0"
 netutils = "^1.0.0"
-ipfabric-diagrams = "^1.2.7"
-ipfabric = "^0"
+ipfabric-diagrams = "5.*"
```

Comment:
If updating pyproject.toml please run `poetry lock` to regenerate the lock file.

Reply:
I can't because I can't do a poetry install on any of my Windows machines

Reply:
I'll regenerate the lock file.